pyquil/tests/test_parser.py

```diff
@@ -391,14 +391,18 @@ def test_empty_program():
 
 def test_def_circuit():
     defcircuit = """
-DEFCIRCUIT bell a b:
+DEFCIRCUIT custom(%theta, %phi) a b:
     H a
     CNOT a b
+    RX(%theta) a
+    RZ(%phi) b
 """.strip()
     defcircuit_no_qubits = """
-DEFCIRCUIT bell:
-    H 0
-    CNOT 0 1
+DEFCIRCUIT custom(%theta, %phi) a b:
```

Comment:
The addition of parameters is good, but it looks like this no longer tests the case where `DEFCIRCUIT` has no qubits.

Reply:
Oops! Good ole copy/paste error. Good catch!

Reply:
Fixed